db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+category1 = Category.create(name: 'Coffee')
+category2 = Category.create(name: 'Brewing')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.